signin.js

```diff
@@ -0,0 +1,57 @@
+
+import {
+  errorBoxOnOff,
+  showError,
+  hideError,
+  checkisValidEmail
+} from "./src/sign.js";
+
+
+function focusOutCheck(event){
+  const { value, id } = event.target;
+  if (id === 'email') {
+    if (value === "") {
+      showError(id, '이메일을 입력해주세요.');
+    } else if (!checkisValidEmail(value)) {
+      showError(id, '올바른 이메일 주소가 아닙니다.');
+    } else {
+      hideError(id);
+    }
+  } else {
+    if (value === "") {
+      showError(id, '비밀번호를 입력해주세요.');
+    } else {
+      hideError(id);
+    }
+  }
+}
+
+
+function loginProcess() {
+  const emailValue = document.querySelector('#email').value
+  const passwordValue = document.querySelector('#password').value
+  if (emailValue !== 'test@codeit.com') {
+    showError('email', '이메일을 확인해 주세요.');
+  } else {
+    hideError('email');
+  }
+
+  if (passwordValue !== 'codeit101') {
+    showError('password', '비밀번호를 확인해 주세요.');
+  } else {
+    hideError('password');
+  }
+
+  if (emailValue === 'test@codeit.com' && passwordValue === 'codeit101') {
+    location.href = '/folder.html';
+  }
+}
+
+
+document.querySelector(`#email`).addEventListener('focusout',  focusOutCheck)
+document.querySelector(`#password`).addEventListener('focusout',  focusOutCheck)
+document.querySelector(`#signForm`).addEventListener('submit', function(event) {
```

Comment:
```suggestion
document.querySelector(`#signForm`).addEventListener('submit', (event) => {
```

만약 익명함수를 사용할거라면 화살표 함수를 써보는건 어떨까요?
함수 선언문을 사용하면 this 바인딩 예측이 힘들어지고 함수 선언문을 기명으로 사용하면 사용처가 없어 기명으로한 의미가 없어지게 됩니다.
하지만 화살표 함수를 사용하게 되면 렉시컬 this로 동작해서 this 예측이 훨씬 쉬워지고 무조건 표현식으로만 사용해야하기 때문에 익명 함수로서 많이 사용되고 있어 권장드립니다.